Sources/Testing/Issues/Issue.swift

```diff
@@ -175,11 +175,9 @@ extension Issue: CustomStringConvertible, CustomDebugStringConvertible {
 /// In the future, when our minimum deployment target supports casting a value
 /// to a constrained existential type ([SE-0353](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0353-constrained-existential-types.md#effect-on-abi-stability)),
 /// we can remove this protocol and cast to `RangeExpression<Int>` instead.
-private protocol _RangeExpressionOverIntValues: RangeExpression where Bound == Int {}
+private protocol _RangeExpressionOverIntValues: RangeExpression & Sequence where Bound == Int, Element == Int {}
```

Comment:
Would it be stronger / more precise to constrain such that `Element == Bound`?
```suggestion
private protocol _RangeExpressionOverIntValues: RangeExpression & Sequence where Bound == Int, Element == Bound {}
```